src/common/editable/index.js

```diff
@@ -137,6 +136,7 @@ class Editable extends Component {
       this._closeEdition()
     } catch (error) {
       this.setState({
+        // `error` may be undefined if this._save() opens a modal and the user closes it
```

Comment:
Less specific: say that the `error` can be `undefined` if the action has been cancelled.